app/controllers/users_controller.rb

```diff
@@ -1,4 +1,5 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!, only: [:show]
```

Comment:
- [ ] **[OPTIONAL]** Since you may add the authenticate helper in the application controller. You may need to remove this one here for it to work.